### PR TITLE
Release/0.1.4

### PR DESCRIPTION
**Pull request by colmsnowplow**
Just a quick bugfix release to update demo App and address this issue: https://github.com/snowplow-incubator/snowplow-react-native-tracker/issues/105

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,8 +15,8 @@
   ],
   "license": "Apache-2.0",
   "peerDependencies": {
-    "react-native": "^0.59.8",
-    "react": "^16.8.6"
+    "react-native": ">=0.59.8",
```
Comment (AlexBenny) [inline package.json:18]: I really dislike the way the manage versioning. Technically it's impossible to know whether there are breaking changes 🤦.

**Review by AlexBenny — APPROVED** [review]
Looks good! 👍 🚢

File: package.json (hunk 1 above)
Comment (colmsnowplow) [reply package.json:18]: Yeah I don't really like it either, I think this is why I fixed to major version before.

**Comment by colmsnowplow** [comment]
Made a mistake in publishing and so had to change version to 0.1.5. Closing this in favour of https://github.com/snowplow-incubator/snowplow-react-native-tracker/pull/108 because of that.

**Pull request closed without merging**